test/pipetool.uts

```diff
@@ -9,13 +9,13 @@
 s = PeriodicSource("hello", 1, name="src")
 d1 = Drain(name="d1")
 c = ConsoleSink(name="c")
-tf = TransformDrain(lambda x:"Got %r" % x)
+tf = TransformDrain(lambda x: "Got %s" % x)
 t = TermSink(name="PipeToolsPeriodicTest", keepterm=False)
 s > d1 > c
 d1 > tf > t
 
 p = PipeEngine(s)
-p.graph(type="png",target="> /tmp/pipe.png")
+p.graph(type="png",target="-o"+get_temp_file(autoext="png"))
```

Comment:
Unless I'm wrong, this is no longer needed. Could you remove this?

Reply:
Right

Reply:
Does this work under Windows?